mnefun/_mnefun.py

```diff
@@ -1818,7 +1817,11 @@ def do_preprocessing_combined(p, subjects, run_indices):
             n_jobs=p.n_jobs_fir, n_jobs_resample=p.n_jobs_resample,
             projs=projs, bad_file=bad_file, disp_files=p.disp_files,
             method='fft', filter_length=p.filter_length, force_bads=False,
-            l_trans=p.hp_trans, h_trans=p.lp_trans)
+            l_trans=p.hp_trans, h_trans=p.lp_trans, eeg_ref=False)
+        for ii in raw_orig.info['projs']:
+            if ii['desc'] == 'Average EEG reference':
+                eeg_ref_proj = ii
+                raw_orig.add_proj(eeg_ref_proj).apply_proj()
 
```

Comment:
This also works with my dataset, did not test with example data b/c I am not sure if 
`for ii in raw_orig.info['projs']:
             if ii['desc'] == 'Average EEG reference':
                 eeg_ref_proj = ii
                 raw_orig.add_proj(eeg_ref_proj).apply_proj()`

is necessary; seems like `raw_orig.apply_proj()` should suffice after the read?